verilog/formatting/formatter.cc

```diff
@@ -230,15 +230,265 @@ static void DeterminePartitionExpansion(partition_node_type* node,
       break;
     }
     case PartitionPolicyEnum::kFitOnLineElseExpand: {
-      if (verible::FitsOnLine(uwline, style)) {
+      if (verible::FitsOnLine(uwline, style).fits) {
         VLOG(3) << "Fits, un-expanding.";
         node_view.Unexpand();
       } else {
         VLOG(3) << "Does not fit, expanding.";
         node_view.Expand();
       }
+      break;
+    }
+    default: {
+      break;
+    }
+  }
+}
+
+//
+// This function formats expanded_lines in a simple greedy manner
+// that is by appending consecutive lines and wrapping only when necessary to stay
+// under the column limit.
```

Comment:
Explain the significance of `expanded_lines[0]` as the line that establishes an indentation/alignment position (when `expanded_lines[1]` is appended).

Reply:
Done